### PR TITLE
update sdk document and chunk

### PR DESCRIPTION
**Pull request by JobSmithManipulation**
### What problem does this PR solve?

_Briefly describe what this PR aims to solve. Include background context that will help reviewers understand the purpose of the PR._

### Type of change

- [x] New Feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by JinHai-CN** [comment]
pdf file is too big for code repository

File: sdk/python/test/t_document.py [hunk 1]
```diff
@@ -138,7 +139,131 @@ def test_delete_documents_in_dataset_with_success(self):
...
+        cancel_result = rag.async_cancel_parse_documents(ids)
+        assert cancel_result is None or isinstance(cancel_result, type(None)), "Failed to cancel document parsing"
+        print("Async bulk parsing cancelled")
+
+    def test_parse_document_and_chunk_list(self):
+        rag = RAGFlow(API_KEY, HOST_ADDRESS)
+        ds = rag.create_dataset(name="God7")
+        name='story.txt'
+        path = 'test_data/story.txt'
+        # name = "Test Document rag.txt"
```
Comment (KevinHuSh) [inline sdk/python/test/t_document.py:233]:
```suggestion
```

File: sdk/python/ragflow/modules/chunk.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+from .base import Base
+
+
+class Chunk(Base):
+    def __init__(self, rag, res_dict):
+        # 初始化类的属性
```
Comment (KevinHuSh) [inline sdk/python/ragflow/modules/chunk.py:6]:
```suggestion
```

File: sdk/python/test/t_document.py [hunk 3]
```diff
@@ -138,7 +139,130 @@ def test_delete_documents_in_dataset_with_success(self):
...
+
+    def test_parse_document_and_chunk_list(self):
+        rag = RAGFlow(API_KEY, HOST_ADDRESS)
+        ds = rag.create_dataset(name="God7")
+        name='story.txt'
+        path = 'test_data/story.txt'
+        # blob = " Sample document content for rag test66. rag wonderful apple os documents apps. Sample document content for rag test66. rag wonderful apple os documents apps.Sample document content for rag test66. rag wonderful apple os documents apps.Sample document content for rag test66. rag wonderful apple os documents apps. Sample document content for rag test66. rag wonderful apple os documents apps. Sample document content for rag test66. rag wonderful apple os documents apps. Sample document content for rag test66. rag wonderful apple os documents apps. Sample document content for rag test66. rag wonderful apple os documents apps. Sample document content for rag test66. rag wonderful apple os documents apps.  Sample document content for rag test66. rag wonderful apple os documents apps. Sample document content for rag test66. rag wonderful apple os documents apps. Sample document content for rag test66. rag wonderful apple os documents apps."
```
Comment (KevinHuSh) [inline sdk/python/test/t_document.py:234]:
```suggestion
```

File: sdk/python/ragflow/modules/document.py [hunk 4]
```diff
@@ -73,3 +76,121 @@ def download(self) -> bytes:
...
+                return chunks
+            else:
+                raise Exception(f"Error fetching chunks: {res_data.get('retmsg')}")
+        else:
+            raise Exception(f"API request failed with status code {res.status_code}")
+
+    def add_chunk(self, content: str):
+        res = self.post('/doc/chunk/create', {"doc_id": self.id, "content_with_weight":content})
+
+        # 假设返回的 response 包含 chunk 的信息
```
Comment (KevinHuSh) [inline sdk/python/ragflow/modules/document.py:191]:
```suggestion
```

File: sdk/python/ragflow/modules/document.py [hunk 5]
```diff
@@ -73,3 +76,120 @@ def download(self) -> bytes:
...
+                raise Exception(f"Error fetching chunks: {res_data.get('retmsg')}")
+        else:
+            raise Exception(f"API request failed with status code {res.status_code}")
+
+    def add_chunk(self, content: str):
+        res = self.post('/doc/chunk/create', {"doc_id": self.id, "content_with_weight":content})
+
+        if res.status_code == 200:
+            chunk_data = res.json()
+            return Chunk(self.rag,chunk_data)  # 假设有一个 Chunk 类来处理 chunk 对象
```
Comment (KevinHuSh) [inline sdk/python/ragflow/modules/document.py:194]:
```suggestion
            return Chunk(self.rag,chunk_data)
```

**Pull request merged**